doc/examples/segmentation/plot_chanvese.py

```diff
@@ -0,0 +1,25 @@
+""" This is an example to illustrate the us of the Chan-Vese Algorithm.
+"""
+
+from skimage.data import camera
+from skimage.segmentation.chanvese import chan_vese
+import numpy as np
+import matplotlib.pyplot as plt
+
+def _cv_main():
+    U0 = camera().astype('float')/255.
+    print np.max(U0)
```

Comment:
This will fail in Python 3. Please, add `from __future__ import print_function` to imports and use `print(%)` everywhere instead of `print %`.